specification/cognitiveservices/ContentSafetyInternal/tspconfig.yaml

```diff
@@ -0,0 +1,31 @@
+parameters:
+  service-dir:
+    default: "sdk/contentsafetyinternal"
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    emitter-output-dir: "{project-root}/../"
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/contentsafetyinternal.json"
+    azure-resource-provider-folder: "data-plane"
+    examples-directory: examples
+  "@azure-tools/typespec-python":
+    package-dir: "azure-ai-contentsafetyinternal"
+    package-name: "{package-dir}"
+    package-mode: dataplane
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.AI.ContentSafetyInternal"
+    namespace: "{package-dir}"
+    clear-output-folder: true
+    model-namespace: false
+  # "@azure-tools/typespec-ts":
+  #   package-dir: "contentsafety-rest"
+  #   generateMetadata: true
+  #   generateTest: true
+  #   azureSdkForJs: true
+  #   packageDetails:
+  #     name: "@azure-rest/contentsafety-rest"
+  #     description: "ContentSafety Service"
+  # "@azure-tools/typespec-java":
+  #   package-dir: "contentsafety"
+  #   namespace: com.contentsafety
```

Comment:
It is better to open Js/Java SDK generaion.

Reply:
Java PR https://github.com/Azure/azure-rest-api-specs/pull/25307